iOS/UZTickets/src/core/UZTTicketInfo.h

```diff
@@ -0,0 +1,102 @@
+//
+//  UZTTicketInfo.h
+//  UZTickets
+//
+//  Created by Алексей Демедецкий on 01.06.15.
+//  Copyright (c) 2015 DAloG ltd. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
You don't actually need to use the @import keyword. If you opt-in to using modules, all #import and #include directives are mapped to use @import automatically. That means that you don't have to change your source code (or the source code of libraries that you download from elsewhere). Supposedly using modules improves the build performance too, especially if you haven't been using PCHs well or if your project has many small source files.

> http://stackoverflow.com/questions/18947516/import-vs-import-ios-7